src/utilities/renderer-helper.ts

```diff
@@ -4,21 +4,20 @@ import Role from '../features/permissions/roles';
 import * as _ from 'lodash';
 import * as Joi from '@hapi/joi';
 import 'joi-extract-type';
-import * as FormData from 'form-data';
 import { isAxiosError } from './axios-helper';
 import logger from './logger';
 import NotFoundError from '../exceptions/not-found-error';
 import WrappedError from '../exceptions/wrapped-error';
 import { RederlyExtendedJoi } from '../extensions/rederly-extended-joi';
 import urljoin = require('url-join');
+import formHelper, { unmergeStrategies } from './form-helper';
 
 const rendererAxios = axios.create({
     baseURL: configurations.renderer.url,
     responseType: 'json',
     timeout: configurations.renderer.requestTimeout
 });
 
-// TODO switch over to new endpoint
```

Comment:
Why did you remove this TODO, it still needs to be done 🤣 
/rendered needs to become /render-api
haven't had time to debug why it doesn't work correctly